demos/system_reference-AMD-Xilinx.rst

```diff
@@ -0,0 +1,10 @@
+.. _demos-AMD--work-label:
+
+==================================================
+System Reference Samples and Demos on AMD platform
```

Comment:
Will this change in a later commit to something like "System Reference Samples and Demos on Xilinx Zynq UltraScale+ MPSoC ZCU102" to mirror the "System Reference Samples and Demos on STM32MP157C/F-DK2 board" entry in ./system_reference-ST.rst?

Reply:
Yes I think so.
%s/AMD platform/AMD-Xilinx ZynqMP zcu102 platform/ should be enough.

Reply:
I'll let @TanmayShah-xilinx complete this file